src/System/Metrics/Network/Acceptor.hs

```diff
@@ -79,6 +79,7 @@ doListenToForwarder snocket address timeLimits app = do
   _ <- async $ cleanNetworkMutableState networkState
   withServerNode
     snocket
+    (\_ _ -> return ())
```

Comment:
@bolt12 I think we don't need to expose this argument at all for `withServerNode`, that's runs non P2P server anyway.